lib/site_prism/waiter.rb

```diff
@@ -2,33 +2,18 @@
 
 module SitePrism
   class Waiter
-    class << self
-      def wait_until_true(wait_time = Capybara.default_max_wait_time)
-        start_time = Time.now
+    SLEEP_DURATION = 0.05
```

Comment:
Style pref - Can we use a private method, instead of a constant

Reply:
I've changed the constant to a method. However, I don't see the need to make it private. Checking the waiter's time value from outside seems fine to me.